addons/knobs/src/base.js

```diff
@@ -46,7 +47,14 @@ export function object(name, value) {
 }
 
 export function select(name, options, value) {
-  return manager.knob(name, { type: 'select', options, value });
+  return deprecate(
+    manager.knob(name, { type: 'select', options, value }),
+    'in v4 keys/values of the options argument are reversed'
+  );
+}
```

Comment:
For some reason, it throws in unit tests:
```
TypeError: Object prototype may only be an Object or null: apple
        at Function.setPrototypeOf (<anonymous>)

      74 | 
      75 | function select(name, options, value) {
    > 76 |   return (0, _utilDeprecate2.default)(manager.knob(name, { type: 'select', options: options, value: value }), 'in v4 keys/values of the options argument are reversed');
      77 | }
      78 | 
      79 | function selectV2(name, options, value) {
      
      at select (addons/knobs/dist/base.js:76:38)
```
https://circleci.com/gh/storybooks/storybook/33655?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
I was doing it wrong :)

fixed this up and added selectV2 example